samples/CMakeLists.txt

```diff
@@ -109,6 +110,13 @@ if (ICPX_EXECUTABLE)
     sycl_hip_interop
     hip_sycl_interop
     hip_sycl_interop_no_buffers)
+
+  # the ENABLE_OMP_OFFLOAD is only required to unhide MKL::sycl in intel's MKLConfig.cmake file
+  set(ENABLE_OMP_OFFLOAD ON)
+  set(MKL_THREADING sequential)
+  include(../cmake/FindMKL.cmake)
+  find_package(MKL REQUIRED CONFIG PATHS /opt/intel/oneapi/mkl/latest/lib/cmake/mkl)
```

Comment:
Should be moved to findMKL.cmake

Reply:
moved it into `cmake/mkl_and_icpx.cmake` which looks for both MKL and icpx.